frontend/src/models/index.tsx

```diff
@@ -152,6 +156,10 @@ export type Settings = {
   };
   updatedAt?: Date;
   navbarTitle?: string;
+  topicAreaLabels?: {
+    singular: string;
+    plural: string;
+  };
```

Comment:
I don't think we should make this property optional. Remove the `?`.  I noticed that you added a default value in the `SetttingsProvider` file so we can assume that this property will always be present.

Reply:
Agreed. Complete